src/Spatial/Euclidean/Point3D.cs

```diff
@@ -226,6 +226,16 @@ public static Point3D IntersectionOf(Plane plane, Ray3D ray)
             return plane.IntersectionWith(ray);
         }
 
+        public static Point3D operator +(Vector3D v, Point3D p)
```

Comment:
My thought here is that addition is commutative, and so to have the order matter diverges from the math itself. I will separate this out so that it doesn't hold up the PR.